tests/plugins/test_orchestrate_build.py

```diff
@@ -823,3 +831,72 @@ def test_orchestrate_build_failed_to_list_builds(tmpdir, fail_at):
             assert 'All clusters for platform x86_64 are unreachable' in str(exc)
         elif fail_at == 'build_canceled':
             assert 'BuildCanceledException()' in str(exc)
+
+
+@pytest.mark.parametrize('is_auto', [
+    True,
+    False
+])
+def test_orchestrate_build_worker_build_kwargs(tmpdir, caplog, is_auto):
+    workflow = mock_workflow(tmpdir)
+    expected_kwargs = {
+        'git_uri': SOURCE['uri'],
+        'git_ref': 'master',
+        'git_branch': 'master',
+        'user': 'bacon',
+        'is_auto': is_auto,
+        'platform': 'x86_64',
+        'release': '10',
+        'arrangement_version': 1
+    }
+    mock_osbs(worker_expect=expected_kwargs)
+    mock_reactor_config(tmpdir)
+
+    plugin_args = {
+        'platforms': ['x86_64'],
+        'build_kwargs': make_worker_build_kwargs(),
+        'worker_build_image': 'fedora:latest',
+        'osbs_client_config': str(tmpdir),
+    }
+
+    runner = BuildStepPluginsRunner(
+        workflow.builder.tasker,
+        workflow,
+        [{
+            'name': OrchestrateBuildPlugin.key,
+            'args': plugin_args,
+        }]
+    )
+    workflow.prebuild_results[CheckAndSetRebuildPlugin.key] = is_auto
+
+    build_result = runner.run()
+    assert not build_result.is_failed()
+
+    assert (build_result.annotations == {
+        'worker-builds': {
+            'x86_64': {
+                'build': {
+                    'build-name': 'worker-build-x86_64',
+                    'cluster-url': 'https://worker_x86_64.com/',
+                    'namespace': 'worker_x86_64_namespace'
+                },
+                'digests': [],
+                'plugins-metadata': {}
+            }
+        }
+    })
+
+    assert (build_result.labels == {})
+
+    build_info = get_worker_build_info(workflow, 'x86_64')
+    assert build_info.osbs
+
+    for record in caplog.records():
+        if not record.name.startswith("atomic_reactor"):
+            continue
+
+        assert hasattr(record, 'arch')
+        if record.funcName == 'watch_logs':
+            assert record.arch == 'x86_64'
+        else:
+            assert record.arch == '-'
```

Comment:
ok removed the same things, which were in different test and kept just the kwargs check